tests/dummy/app/router.js

```diff
@@ -7,7 +7,6 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('demo');
```

Comment:
Is this change intentional?  I can't recall if we have anything meaningful on this page...

Reply:
hmmmm. I thought it was superfluous, but now that I look again I should probably add it back.